taxi/views.py

```diff
@@ -1,3 +1,17 @@
+from django.http import HttpResponse, HttpRequest
+
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Car, Driver, Manufacturer
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+    context = {
+        "num_drivers": num_drivers,
+        "num_manufacturers": num_manufacturers,
+        "num_cars": num_cars
+    }
+    return render(request, "taxi/index.html", context=context, )
```

Comment:
There is no need to leave trailing coma
```suggestion
    return render(request, "taxi/index.html", context=context)
```